README.md

```diff
@@ -14,3 +16,47 @@ This lib depends on libgraphqlparser (in submodule in this repository).
 
 You need to build and install this libgraphqlparser binary on your system before using this.
 See [here](https://github.com/graphql/libgraphqlparser) for more details.
+
+## Roadmap - Road to v1
+
+Based on our experience to managing GraphQL API in production for more than one year. We decided to work on a brand new GraphQL engine.
+
+Here are the subjects we are working on:
+
+- [x] Define Open Source guidelines (Contributing, Code of conduct, Issue Template, PR Template).
+- [x] Build communication interface between [libgraphqlparser](https://github.com/graphql/libgraphqlparser) & Tartiflette through [CFFI](https://cffi.readthedocs.io).
+- [x] Build a `Parser` which parse a GraphQL Query and creates a list of Asynchronous Tasks.
+- [x] Build an `Executor` which execute the Async Tasks list created by the `GraphQL Query Parser`.
+- [x] Build a `Parser` which parse a [Schema Definition Language](https://github.com/facebook/graphql/blob/master/spec/Section%202%20--%20Language.md) and creates the associated types in Python object.
+- [x] Bind the Types specified in the SDL to the `Executor`.
+- [ ] (Executor) Use `Field` instead `Node`.
+- [ ] (Doc) Describe the API that will be used by the Tartiflette users.
+  - [ ] (Doc) API - Engine constructor.
+  - [ ] (Doc) API - How to declare custom directive in the Engine Constructor.
+  - [ ] (Doc) API - How to declare middleware on resolver.
+  - [ ] (Doc) API - Executor.
+  - [ ] (Doc) API - Resolver.
+  - [ ] (Doc) API - Resolver - `Info` parameter.
+- [ ] (Introspection) Implement the `__type` Field.
+- [ ] (Introspection) Implement the `__schema`Field.
+- [ ] (Resolver) Algorithm to apply [default resolver](https://github.com/dailymotion/tartiflette/blob/master/tartiflette/parser/visitor.py#L18)
+- [ ] (Executor) Typing [resolver outputs](https://github.com/dailymotion/tartiflette/blob/master/tartiflette/parser/visitor.py#L191)
+- [ ] (Executor) Error management
+- [ ] (Executor) Abstract and Compound Types: Scalar, Unions, Interfaces, …
```

Comment:
Keep just Unions, Interfaces: Scalars are not abstract nor compound, my mistake in formulating this